cmk/special_agents/agent_prism.py

```diff
@@ -55,117 +40,120 @@ def parse_arguments(argv: Sequence[str] | None) -> Args:
     return parser.parse_args(argv)
 
 
-# TODO: get rid of CSV and write JSON
 def output_containers(requester: Requester) -> None:
     LOGGING.debug("do request..")
     obj = requester.get("containers")
     LOGGING.debug("got %d containers", len(obj["entities"]))
 
     write_title("containers")
-    writer = gen_csv_writer()
-    writer.writerow(["name", "usage", "capacity"])
-    for entity in obj["entities"]:
-        writer.writerow(
-            [
-                entity["name"],
-                entity["usageStats"]["storage.user_usage_bytes"],
-                entity["usageStats"]["storage.user_capacity_bytes"],
-            ]
-        )
-
+    print(obj)
```

Comment:
pasting raw python objects as a serialization format is not great. Please rather use the json module. Those are more guaranteed to be safe to parse.